frameworks/elastic/tests/config.py

```diff
@@ -241,7 +244,10 @@ def get_document(index_name, index_type, doc_id, service_name=PACKAGE_NAME):
 
 
 def _curl_api(service_name, method, role="master"):
-    host = "http://" + sdk_hosts.autoip_host(service_name, "{}-0-node".format(role), _master_zero_http_port(service_name))
+    host = 'http://'
+    if ELASTIC_TLS_ENABLED:
+        host = 'https://'
+    host = host + sdk_hosts.autoip_host(service_name, "{}-0-node".format(role), _master_zero_http_port(service_name))
     return ("curl -X{} -s -u elastic:changeme '" + host).format(method)
```

Comment:
This is outside the scope of this PR, but I assume that the reason we use `curl` instead of (e.g.) the `requests` package is that we need to run these commands on the master? Is there a public endpoint that we could use for testing instead?